lib/data/utils/capitalize.dart

```diff
@@ -0,0 +1,8 @@
+String capitalize(String str) {
```

Comment:
we can add it as string extensions. we can create extensions.dart and reuse that for all extensions

https://dart.dev/language/extension-methods

Reply:
Done.